examples/progressive-web-app/pages/api/hello.js

```diff
@@ -4,4 +4,4 @@ const hello = (req, res) => {
   res.status(200).json({ name: 'John Doe' })
 }
 
-export default hello
+export default hello
```

Comment:
It appears there are unrelated changes to files other than the Tailwind example - could we revert all of those?

Reply:
Yes sure